documents/model.go

```diff
@@ -117,8 +117,8 @@ type Model interface {
 	// IsDIDCollaborator returns true if the did is a collaborator of the document
 	IsDIDCollaborator(did identity.DID) (bool, error)
 
-	// AddAttribute adds a custom attribute to the model with the given value. If an attribute with the given name already exists, it's updated.
-	AddAttributes(attrs ...Attribute) error
+	// AddAttributes adds a custom attribute to the model with the given value. If an attribute with the given name already exists, it's updated.
+	AddAttributes(ca CollaboratorsAccess, prepareNewVersion bool, attrs ...Attribute) error
```

Comment:
Add a bit of explanation about what PrepareNewVersion does. Same for DeleteAttributes